### ISSUE TITLE
[BUG][TGL][SDW] IPC timed out for 0x60060000 size 12 occured when doing Dmic(0,3) pause_release stress test.

### ISSUE THREAD

**Issue by sinahuang** [thesofproject/linux#1519]
**Describe the bug**
IPC timed out for 0x60060000 size 12 occured in legacy mode when doing dmic(0,3)/speaker pause_release stress test. 
Issue only can be reproduced on Dmic(0,3), on DMIC(0,4) issue cannot be reproduced.

**To Reproduce**
1.Boot up system;
2.Run "./sof-test-pause-release.sh -t pcm3c";

Scripts:
[sof-test-pause-release.txt](https://github.com/thesofproject/linux/files/3867602/sof-test-pause-release.txt)

**Reproduce rate**
Tried on Dmic(0,3):4/300

**Expected result**
Dmic/speaker pause_release test should be work normally. 

**Actual result**
IPC timed out for 0x60060000 size 12 occured when doing  dmic/speaker pause_release stress test.

Dmesg 
```
[  246.477162] sof-audio-pci 0000:00:1f.3: ipc rx: 0x90020000: GLB_TRACE_MSG
[  246.477200] sof-audio-pci 0000:00:1f.3: ipc rx done: 0x90020000: GLB_TRACE_MSG
[  247.112165]  Speaker: ASoC: pre trigger FE Speaker cmd 3
[  247.112168] sof-audio-pci 0000:00:1f.3: pcm: trigger stream 2 dir 0 cmd 3
[  247.112170] sof-audio-pci 0000:00:1f.3: ipc tx: 0x60060000: GLB_STREAM_MSG: TRIG_PAUSE
[  247.619897] sof-audio-pci 0000:00:1f.3: error: ipc timed out for 0x60060000 size 12
[  247.619901] sof-audio-pci 0000:00:1f.3: info: preventing DSP entering D3 state to preserve context
[  247.619905] sof-audio-pci 0000:00:1f.3: status: fw entered - code 00000005
[  247.620320] sof-audio-pci 0000:00:1f.3: error: unexpected fault 0x00000000 trace 0x00004000
[  247.620330] sof-audio-pci 0000:00:1f.3: error: hda irq intsts 0x80000080 intlctl 0xc0000081 rirb 00
[  247.620331] sof-audio-pci 0000:00:1f.3: error: dsp irq ppsts 0x00000000 adspis 0x00000001
[  247.620344] sof-audio-pci 0000:00:1f.3: error: host status 0x80000000 dsp status 0x90020000 mask 0x00000003
[  247.620346] sof-audio-pci 0000:00:1f.3: error: waking up any trace sleepers
[  247.620367] sof-audio-pci 0000:00:1f.3: FW Poll Status: reg=0x20140000 successful
[  247.620378]  Speaker: ASoC: trigger FE cmd: 3 failed: -110
[  247.620380]  Speaker: ASoC: post trigger FE Speaker cmd 4
[  247.620381] sof-audio-pci 0000:00:1f.3: pcm: trigger stream 2 dir 0 cmd 4
[  247.620478] sof-audio-pci 0000:00:1f.3: ipc rx: 0x90020000: GLB_TRACE_MSG
[  247.620510] sof-audio-pci 0000:00:1f.3: ipc rx done: 0x90020000: GLB_TRACE_MSG
```

**Test recipe**

Kernel:https://github.com/thesofproject/linux/tree/integration/soundwire-latest commit: 4cd6497
FW: https://github.com/thesofproject/sof/commits/master commit: 63a5529
Tplg: Same with TGL FW branch commit: sof-tgl-rt711-rt1308-nohdmi.tplg rename to sof-tgl-rt711-rt1308.tplg
Platform:TGL RVP with onboard ACL711 in SDW mod and ALC1308 in I2S mode.

[dmesg.log](https://github.com/thesofproject/linux/files/3867575/dmesg.log)
[sof_trace.log](https://github.com/thesofproject/linux/files/3867576/sof_trace.log)
[sof_error_trace.log](https://github.com/thesofproject/linux/files/3867577/sof_error_trace.log)

**Comment by sinahuang** [comment]
Issue cannot be reproduced on TGL-RVP in legacy mode with soundwire-latest(a370cb0+ PR #1548 ), this issue seems can be fixed after PR #1548 merged.

Test for 800 times.

**Test recipe**

Kernel:https://github.com/thesofproject/linux/tree/integration/soundwire-latest commit: a370cb0 +PR #1548 
FW: https://github.com/thesofproject/sof/commits/master commit: db7671a
Tplg: Same with TGL FW branch commit: sof-tgl-rt711-rt1308-nohdmi.tplg rename to sof-tgl-rt711-rt1308.tplg
Platform:TGL RVP with onboard ACL711 in SDW mod and ALC1308 in I2S mode.

**Comment by sinahuang** [comment]
Issue cannot be reproduced on TGL-RVP with legacy mode. So closed it.

Test for 1000 times.

**Test recipe**

Kernel:https://github.com/thesofproject/linux/tree/integration/soundwire-latest commit: d6078f9
FW: https://github.com/thesofproject/sof/commits/master commit: d740a3e
Tplg: Same with TGL FW branch commit: sof-tgl-rt711-rt1308-nohdmi.tplg rename to sof-tgl-rt711-rt1308.tplg
Platform:TGL RVP with onboard ACL711 in SDW mod and ALC1308 in I2S mode.
